LoopKitUI/Views/Settings Editors/TherapySettingsView.swift

```diff
@@ -180,6 +184,31 @@ extension TherapySettingsView {
         }
     }
     
+    private var basalRatesSection: some View {
+        section(for: .basalRate) {
+            Group {
+                if self.therapySettings.basalRateSchedule != nil && self.viewModel.supportedBasalRates != nil {
+                    ForEach(self.therapySettings.basalRateSchedule!.items, id: \.self) { value in
+                        ScheduleValueItem(time: value.startTime,
+                                          value: value.value,
+                                          unit: .internationalUnitsPerHour,
+                                          guardrail: Guardrail.basalRate(supportedBasalRates: self.viewModel.supportedBasalRates!))
+                    }
+                }
+                if self.viewModel.supportedBasalRates == nil {
+                    Text("NO supportedBasalRates")
+                }
+                if self.therapySettings.basalRateSchedule == nil {
+                    Text("NO basalRateSchedule")
+                }
```

Comment:
Are there expected circumstances under which these conditions will hold true? Looks like it would be either 1) a PumpManager is not yet configured or 2) the basal rate schedule hasn't been set.

Either of these conditions could occur in DIY—we might want to consider what these schedule sections should look like when no schedule items are present. A button with a link to the editor?

Reply:
In this case, though, we'd want to link to the pump setup itself, not the basal rates editor (in DIY, it currently just doesn't let you enter into the basal rates editor without a pump set up). I think the simplest solution could be to have some sort of text pop up above the setting if the pump isn't set up that indicates a pump needs to be set to add basal rates. For the purposes of 510K, I'm not sure this needs to be there, but it would be a good TODO

Reply:
@mpangburn short answer is: no.  TBH I put these in here as a debugging tool, but left them because I thought it might be handy in case the situation came up.  I'll remove them.

Reply:
A bigger conversation needs to be had about how much to have the UI "support" (i.e. be clear to the user when there are) no settings yet.